docs/website/docs/dlt-ecosystem/destinations/staging.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Staging
```

Comment:
cool we have some docs. I'd restructure it:
1. put `staging.md` directly in the `dlt-ecosystem` and explain the general mechanism maybe with one example.
2. put the redshift info into redshift destination same for bigquery